go/teams/chain_parse.go

```diff
@@ -37,6 +39,19 @@ type SCTeamMembers struct {
 	None    *[]SCTeamMember `json:"none,omitempty"`
 }
 
+type SCTeamInvites struct {
```

Comment:
I don't understand why you would want this format. It seems like it makes the checks complicated. What about just a list of `SCTeamInvite`s which are

```
type SCTeamInvite struct {
    ID keybase1.TeamInviteID
    Type string
    Name string
    Role keybase1.TeamRole
}
```